apps/speed.c

```diff
@@ -2545,6 +2559,10 @@ int speed_main(int argc, char **argv)
 
             if (!EVP_MAC_CTX_set_params(loopargs[i].mctx, params))
                 goto skip_hmac; /* Digest not found */
+                /* JMJM: Is skipping the hmac tests the right thing to do?
+                 * wouldn't it be better to fail immediately? i.e. "goto end;"?
+                 * this would be more consistent with the other blocks below.
+                 */
```

Comment:
please let me know your opinion on this.

Reply:
It would be OK to change this if this does not break plain `openssl speed` command on no-md5 builds.

Reply:
okay.  It should be possible to use the macro `OPENSSL_NO_MD5` inside `speed.c` to accommodate those builds.

I will come back to that in a separate PR.

I will push an update and drop my comment.

Reply:
Do `no-md5` builds still exist?  How do I create one?

In `CHANGES.md` it says the `no-md5` config option was dropped in 1.1.0